config/resources/dossier-domain.lisp

```diff
@@ -6,16 +6,18 @@
                 (:is-archived   :boolean  ,(s-prefix "ext:isGearchiveerd"))
                 (:title         :string   ,(s-prefix "dct:title"))
                 (:confidential  :boolean  ,(s-prefix "ext:vertrouwelijk")))
-  :has-many `((subcase          :via      ,(s-prefix "dossier:doorloopt")
-                                :as "subcases")
-              (piece            :via      ,(s-prefix "dossier:Dossier.bestaatUit")
-                                :as "pieces")
-              (publication-flow :via      ,(s-prefix "dossier:behandelt")
-                                :inverse t
-                                :as "publication-flows")
-              (sign-flow        :via      ,(s-prefix "sign:behandeltDossier")
-                                :inverse t
-                                :as "sign-flows")
+  :has-many `((subcase           :via      ,(s-prefix "dossier:doorloopt")
+                                 :as "subcases")
+              (piece             :via      ,(s-prefix "dossier:Dossier.bestaatUit")
+                                 :as "pieces")
+              (publication-flow  :via      ,(s-prefix "dossier:behandelt")
+                                 :inverse t
+                                 :as "publication-flows")
+              (sign-flow         :via      ,(s-prefix "sign:behandeltDossier")
+                                 :inverse t
+                                 :as "sign-flows")
+              (government-fields :via ,(s-prefix "besluitvorming:beleidsgebied") ;; NOTE: temporary name for relationship
+                                 :as "government-fields")
```

Comment:
De reden van die skos:Concepten is dat we die dan vb hier zouden gebruiken.
Want nu heb je enkel een lijst `government-fields`, terwijl we hier een gemengde lijst willen van fields en domains.
```
(concept    :via ,(s-prefix "ext:beleidsgebied") ;; NOTE: temporary name for relationship
                                 :as "government-areas")
```

Reply:
Ook gebruiken we zomaar geen `besluitvorming` als de relatie niet bestaat in het oslo model, dus `ext` is dan de juiste prefix.

Reply:
Okidoki. ik had het anders begrepen. Ik ga het herwerken